frontend/package.json

```diff
@@ -51,6 +51,7 @@
     "@fullcalendar/timeline": "^5.11.0",
     "@hookform/resolvers": "^2.9.3",
     "@iconify/react": "^3.2.2",
+    "@mui/icons-material": "^5.10.9",
```

Comment:
Do we need this? I think Iconify has all the mui icons.

Reply:
agree.